index.html

```diff
@@ -1,19 +1,56 @@
-<!doctype html>
+<!DOCTYPE html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
+  <head>
+    <meta charset="utf-8" />
 
-  <title>My Resume</title>
+    <title>CJ's Resume</title>
 
-  <link rel="stylesheet" href="css/index.css">
+    <link rel="stylesheet" href="css/index.css" />
 
-  <!--[if lt IE 9]>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
-</head>
+    <!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+  </head>
+  <header>
+    <nav>
+      <a href="#">HOME</a>
+      <a href="#">INTRO</a>
+      <a href="#">SKILLS</a>
+      <a href="#">WORK HISTORY</a>
+      <a href="#">CONTACT</a>
+    </nav>
+  </header>
 
-<body>
-  <h1>My Resume</h1>
-</body>
-</html>
\ No newline at end of file
+  <body>
+    <div class="intro">
+      <h1>Carl'averis Jackson</h1>
+      <h2>Graphic Designer/ Web-Dev</h2>
+      <img />
```

Comment:
If you are going to have an empty `<img>` element, make sure you at least write `alt="whatever message"`